python/tvm/relay/frontend/onnx.py

```diff
@@ -719,7 +719,8 @@ def _impl_v1(cls, inputs, attr, params):
             inputs[0] = _op.transpose(inputs[0], axes=(1, 0))
         if not transB:
             inputs[1] = _op.transpose(inputs[1], axes=(1, 0))
-        inputs[0] = _op.nn.batch_flatten(inputs[0])
+        if len(infer_type(inputs[0]).checked_type.shape) != 2:
```

Comment:
Please share the result of `infer_type(inputs[0])` with the one at https://github.com/apache/tvm/blob/96395b46c3899a610d5fcfc8a5da3949a9edb605/python/tvm/relay/frontend/onnx.py#L710